pkgs/development/tools/build-managers/scons/default.nix

```diff
@@ -5,10 +5,6 @@ let
     python = python3;
   };
 in {
-  scons_3_0_1 = (mkScons {
-    version = "3.0.1";
-    sha256 = "0wzid419mlwqw9llrg8gsx4nkzhqy16m4m40r0xnh6cwscw5wir4";
-  }).override { python = python2; };
```

Comment:
Removing without throwing or adding an alias, because the package was not exposed in `all-packages.nix`.